### PR TITLE
Remove ActiveSupport::Inflector

### PR DESCRIPTION
**Pull request by ronin**
This removes dependency on `ActiveSupport::Inflector`. One class that used it was changed to use `Inflecto` instead.

### CODE REVIEW

File: lib/karafka/routing/consumer_mapper.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ module Routing
     #   module MyMapper
     #     def self.call(raw_consumer_group_name)
     #       [
-    #         Karafka::App.config.client_id.to_s.underscope,
```
Comment (mensfeld) [inline lib/karafka/routing/consumer_mapper.rb]: I guess we can't use ```underscore``` directly anymore right?

**Comment by mensfeld** [comment]
Hey @ronin, thank you for your contribution. Everything looks great but just one more idea - maybe we could (while you're still on it) switch to http://dry-rb.org/gems/dry-inflector/ as we tend to use more and more dry-stuff.

File: lib/karafka/routing/consumer_mapper.rb (hunk 1 above)
Comment (ronin) [reply lib/karafka/routing/consumer_mapper.rb]: Yeah. Unless you use Karafka together with Rails application or require ActiveSupport yourself.

Comment (mensfeld) [reply lib/karafka/routing/consumer_mapper.rb]: Ok, so we need to update this example as well.

**Comment by mensfeld** [comment]
Great work btw. The less activesupport the better ;)

**Comment by ronin** [comment]
@mensfeld Sure, I've switched to `dry-inflector` and updated the comment.

**Comment by mensfeld** [comment]
Good afternoon @ronin. Could you please resolve introduced issues: https://app.coditsu.io/karafka/builds/diffs/diffs/96a6c262-783a-4c85-9d49-2c0e3912caaf/offenses/introduced - thanks :)

**Comment by ronin** [comment]
@mensfeld Ok, fixed

**Comment by mensfeld** [comment]
@ronin  thank you once again!

**Pull request merged**